src/x/xrandr.c

```diff
@@ -655,6 +657,12 @@ static bool xrandr_get_monitor_info(ALLEGRO_SYSTEM_XGLX *s, int adapter, ALLEGRO
    mi->y1 = crtc->y;
    mi->x2 = crtc->x + crtc->width;
    mi->y2 = crtc->y + crtc->height;
+#define INCHES_PER_MM 0.039370
```

Comment:
Could you just add this (as _AL_INCHES_PER_MM) to allegro/internal/aintern.h so we don't need to re-define it everywhere?